bin/acceptance-tests.sh

```diff
@@ -6,12 +6,17 @@ set -eo pipefail
 
 # Prep:
 docker-compose --profile acceptance-tests up -d
-WP_PORT=`docker port qm-server | grep "[0-9]+$" -ohE | head -1`
-CHROME_PORT=`docker port qm-chrome | grep "[0-9]+$" -ohE | head -1`
-DATABASE_PORT=`docker port qm-database | grep "[0-9]+$" -ohE | head -1`
+WP_PORT=`docker inspect --type=container --format='{{(index .NetworkSettings.Ports "80/tcp" 0).HostPort}}' qm-server`
+CHROME_PORT=`docker inspect --type=container --format='{{(index .NetworkSettings.Ports "4444/tcp" 0).HostPort}}' qm-chrome`
+DATABASE_PORT=`docker inspect --type=container --format='{{(index .NetworkSettings.Ports "3306/tcp" 0).HostPort}}' qm-database`
 WP_URL="http://host.docker.internal:${WP_PORT}"
 WP="docker-compose run --rm wpcli --url=${WP_URL}"
 
+# Wait for Selenium, the web server, and the database:
+./node_modules/.bin/wait-port -t 10000 $CHROME_PORT
+./node_modules/.bin/wait-port -t 10000 $WP_PORT
+./node_modules/.bin/wait-port -t 10000 $DATABASE_PORT
```

Comment:
@johnbillion Is it likely that Selenium opens its port first?
Aren't the other two faster to boot?

Reply:
Yes probably but in my local testing there's barely any difference. The problem seems to be more with whether Selenium is actually ready, rather than whether its port is responding. I need to investigate it further.